basictracer/tracer.py

```diff
@@ -64,6 +90,12 @@ def start_span(
             # TODO only the first reference is currently used
             parent_ctx = references[0].referenced_context
 
+        # retrieve the active SpanContext
```

Comment:
if we pass `child_of=` kwarg, we ignore the `ScopeManager`, correct?

Reply:
In theory multiple parents will be supported - but currently, only one is used. So, yes, if we pass `child_of`, we ignore the current `Scope`.